tests/test_interactions.py

```diff
@@ -353,122 +354,167 @@ def test_tail_large_negative(self,
                                     seed=42)
 
-        assert interactions.positive_items == {}
+        assert interactions.positive_sets == {}
 
         for _ in range(3):
-            _, negative_samples = interactions[list(range(NUM_USERS_TO_GENERATE))]
+            _, negative_samples = interactions[list(range(NUM_USERS_OR_ITEMS_TO_GENERATE))]
 
-            assert len(negative_samples) == NUM_USERS_TO_GENERATE
+            assert len(negative_samples) == NUM_USERS_OR_ITEMS_TO_GENERATE
 
             for negative_sample in negative_samples:
                 assert len(negative_sample) == NUM_NEGATIVE_SAMPLES
 
     def test_Interactions_approximate_negative_samples_partway_through(
         self,
         ratings_matrix_for_interactions,
+        negative_sample_type,
     ):
         with pytest.warns(UserWarning):
             interactions = Interactions(mat=ratings_matrix_for_interactions,
+                                        negative_sample_type=negative_sample_type,
                                         num_negative_samples=NUM_NEGATIVE_SAMPLES,
                                         max_number_of_samples_to_consider=1,
                                         seed=42)
 
-        assert interactions.positive_items != {}
+        assert interactions.positive_sets != {}
 
         for _ in range(3):
             _, negative_samples = interactions[0]
 
             assert len(negative_samples) == 3
 
-    def test_Interactions_exact_negative_samples(self, ratings_matrix_for_interactions):
+    def test_Interactions_exact_negative_samples(
+        self,
+        ratings_matrix_for_interactions,
+        negative_sample_type,
+    ):
         interactions = Interactions(mat=ratings_matrix_for_interactions,
+                                    negative_sample_type=negative_sample_type,
                                     num_negative_samples=NUM_NEGATIVE_SAMPLES,
                                     max_number_of_samples_to_consider=200,
                                     seed=42)
 
-        assert interactions.positive_items != {}
+        assert interactions.positive_sets != {}
 
         all_negative_samples = list()
         for _ in range(10):
-            _, negative_samples = interactions[0]
+            (_, item_id), negative_samples = interactions[0]
 
             assert len(negative_samples) == NUM_NEGATIVE_SAMPLES
 
             for negative_sample in negative_samples:
-                assert negative_sample.item() not in ratings_matrix_for_interactions[0].nonzero()[0]
+                if negative_sample_type == 'item':
+                    assert negative_sample.item() not in (
+                        ratings_matrix_for_interactions[0].nonzero()[0]
+                    )
+                elif negative_sample_type == 'user':
+                    assert negative_sample.item() not in (
+                        ratings_matrix_for_interactions[:, item_id].nonzero()[0]
```

Comment:
The `item_id` should always be `1` here, right? 

https://github.com/ShopRunner/collie/blob/main/tests/fixtures/interactions_fixtures.py#L24